src/navsat_transform.cpp

```diff
@@ -92,6 +94,18 @@ namespace RobotLocalization
     nh_priv.param("transform_timeout", transform_timeout, 0.0);
     transform_timeout_.fromSec(transform_timeout);
 
+    // Check for deprecated parameters
+    if (nh_priv.param("broadcast_utm_transform", unused, false))
+    {
+      ROS_WARN("navsat_transform, Parameter 'broadcast_utm_transform' has been deprecated. Please use"
```

Comment:
I think we should maybe warn them, but then set the equivalent `broadcast_cartesian_transform` setting anyway. It gets a little more complicated if they do something silly like set `broadcast_cartesian_transform: false` and then `broadcast_utm_transform: true` (or vice-versa), but there's no reason for this to get too complex.

Reply:
Fair point. I've made the change to set the parameters when using the deprecated names.

Reply:
Sorry to be a pain, but I think this will overwrite the value of `broadcast_cartesian_transform_` with _false_ if they _don't_ have `broadcast_utm_transform` set. Maybe change this to `getParam`?

```
if (nh_priv.getParam("broadcast_utm_transform", broadcast_cartesian_transform_))
```

Reply:
Yes, you are absolutely right. I changed that to `getParam()` and verified that it doesn't overwrite the values.